cmd/agent/main.go

```diff
@@ -70,6 +72,7 @@ type GrpcProxyAgentOptions struct {
 	adminServerPort  int
 
 	agentID       string
+	agentAddress  string
```

Comment:
Quibble: I would change this to agentIdentifiers as its more than just the address at this point and it can encode multiple identifiers.

Reply:
Or maybe characteristics or labels.

Reply:
change the `agentAddress` to `agentIdentifiers`